.maintain/build-image.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+target_tag=$1
+
+# build parachain image and standalone image
+docker build --cache-from $AWS_REGISTRY/kilt-parachain/collator:$target_tag --build-arg NODE_TYPE=kilt-parachain -t kiltprotocol/kilt-node:$target_tag .
```

Comment:
I think we should replace all occurrences of `kiltprotocol/*` with the relative env variable.

Reply:
I'll rename that. The idea is, that the build script builds the images with a name X which doesn't really matter how it's called.
The publish script tags X to the docker and AWS names, and pushes those two.